### PR TITLE
Add VERSION file and version number to filename

### PR DESCRIPTION
**Pull request by StefanScherer**
Add a `VERSION` file with the release version to create output file with this version number in its filename.

Connects to #13

### CODE REVIEW

File: test/image_spec.rb [hunk 1]
```diff
@@ -42,10 +39,14 @@
     it "is a HypriotOS" do
       expect(stdout).to contain('HypriotOS')
     end
+
+    it "is not dirty" do
+      expect(stdout).not_to contain('dirty')
```
Comment (StefanScherer) [inline test/image_spec.rb]: Seems that the root fs is "dirty", I have added a test and Travis build 92 https://travis-ci.org/hypriot/image-builder-rpi/builds/104843831 shows that there is a problem.

Refactored the other tests for better readability.

**Comment by StefanScherer** [comment]
@Govinda-Fichtner please review

File: Makefile [hunk 2]
```diff
@@ -1,14 +1,17 @@
...
+	docker run --rm --privileged -v $(shell pwd):/workspace -v /boot:/boot -v /lib/modules:/lib/modules -e VERSION image-builder-rpi
 
 shell: build
-	docker run -ti --privileged -v $(shell pwd):/workspace -v /boot:/boot -v /lib/modules:/lib/modules image-builder-rpi bash
+	docker run -ti --privileged -v $(shell pwd):/workspace -v /boot:/boot -v /lib/modules:/lib/modules -e VERSION image-builder-rpi bash
 
-tag:
-	git tag ${TAG}
-	git push origin ${TAG}
+relese:
```
Comment (Govinda-Fichtner) [inline Makefile]: @StefanScherer This is type, right? Guess it should be "release"

File: builder/build.sh [hunk 3]
```diff
@@ -24,7 +24,8 @@ RAW_IMAGE="rpi-raw.img"
 RAW_IMAGE_VERSION="v0.0.5"
 
 # name of the sd-image we gonna create
-IMAGE_NAME="sd-card-rpi.img"
+IMAGE_VERSION=${VERSION:="v0.0.0"}
+IMAGE_NAME="sd-card-rpi-${IMAGE_VERSION}.img"
```
Comment (DieterReuter) [inline builder/build.sh]: Changing the image file name requires to change the release deployment in [.travis.yml](https://github.com/hypriot/image-builder-rpi/blob/master/.travis.yml#L12) too.

File: VERSION [hunk 4]
```diff
@@ -0,0 +1 @@
+v0.1.0
```
Comment (StefanScherer) [inline VERSION]: I think we should get rid of this VERSION file. The os-rootfs repo works fine without it.

**Pull request merged**